docs/kafka/getting-started-kafka/README.adoc

```diff
@@ -314,7 +314,9 @@ image::sak-create-topic.png[Image of wizard to create a topic]
 
 After you complete the topic setup, the new Kafka topic is listed in the topics table. You can now start producing and consuming messages to and from this topic using services that you connect to this instance.
 
-NOTE: If your topic creation is unsuccessful and you receive an error message, try to create your topic again later. This situation might occur, for example, if your selected cloud provider has a temporary availability problem that affects your Kafka instance.
+NOTE: If you try to create a topic with a number of partitions that would cause the partition limit of the Kafka instance to be exceeded, you see an error message indicating this. For more information about partition limits for Kafka instances, see https://access.redhat.com/articles/5979061[Red Hat OpenShift Streams for Apache Kafka service limits].
```

Comment:
```suggestion
NOTE: If you try to create a topic with a number of partitions that would cause the partition limit of the Kafka instance to be exceeded, you see an error message indicating this. For more information about partition limits for Kafka instances, see https://access.redhat.com/articles/5979061[Red Hat {product-long-kafka} service limits].
```

or maybe

```suggestion
NOTE: If you try to create a topic with a number of partitions that would cause the partition limit of the Kafka instance to be exceeded, you see an error message indicating this. For more information about partition limits for Kafka instances, see https://access.redhat.com/articles/5979061[{product-long-kafka} service limits].
```

or even better, since this link is being added to 3 places here (and may be in other places already?), then the link and name could be pulled out to an attribute, and then:

```suggestion
NOTE: If you try to create a topic with a number of partitions that would cause the partition limit of the Kafka instance to be exceeded, you see an error message indicating this. For more information about partition limits for Kafka instances, see {product-kafka-service-limits-url}.
```